### ISSUE TITLE
Ahhhhhhhh! You have found a bug. 🐞

### ISSUE THREAD

**Issue by nperraud** [SwissDataScienceCenter/renku-python#1165]
## Describe the bug
A clear and concise description.

## Details
*Please verify and redact the details.*

**Renku version:** 0.10.2
**OS:** Linux (#1 SMP Wed Jul 10 08:46:17 EDT 2019)
**Python:** 3.7.3

### Traceback

```
Traceback (most recent call last):
  File "[...]/renku/cli/exception_handler.py", line 119, in main
    result = super().main(*args, **kwargs)
  File "[...]/renku/cli/exception_handler.py", line 90, in main
    return super().main(*args, **kwargs)
  File "[...]/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "[...]/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "[...]/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "[...]/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "[...]/renku/core/commands/client.py", line 103, in new_func
    result = ctx.invoke(method, client, *args, **kwargs)
  File "[...]/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "[...]/renku/cli/update.py", line 149, in update
    outputs = graph.build(revision=revision, can_be_cwl=no_output, paths=paths)
  File "[...]/renku/core/commands/graph.py", line 327, in build
    self.process_dependencies(dependencies, visited=ignore)
  File "[...]/renku/core/commands/graph.py", line 293, in process_dependencies
    activity = processing.client.process_commit(processing.commit)
  File "[...]/renku/core/management/repository.py", line 219, in process_commit
    path=path,
  File "[...]/renku/core/models/cwl/workflow.py", line 72, in create_run
    return WorkflowRun(**kwargs)
  File "<attrs generated init renku.core.models.provenance.activities.WorkflowRun>", line 66, in __init__
    self.__attrs_post_init__()
  File "[...]/renku/core/models/provenance/activities.py", line 714, in __attrs_post_init__
    self.inputs = self.default_inputs()
  File "[...]/renku/core/models/provenance/activities.py", line 453, in default_inputs
    id=usage_id,
  File "[...]/renku/core/models/provenance/qualified.py", line 109, in from_revision
    entity=Entity.from_revision(client, path, revision), **kwargs
  File "[...]/renku/core/models/entities.py", line 137, in from_revision
    client.find_previous_commit(path, revision=revision),
  File "[...]/renku/core/management/repository.py", line 235, in find_previous_commit
    file_commits = list(self.repo.iter_commits(revision, paths=paths))
  File "[...]/git/objects/commit.py", line 277, in _iter_from_process_or_stream
    finalize_process(proc_or_stream)
  File "[...]/git/util.py", line 328, in finalize_process
    proc.wait(**kwargs)
  File "[...]/git/cmd.py", line 408, in wait
    raise GitCommandError(self.args, status, errstr)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(128)
  cmdline: git rev-list 52d413787f13afcd9ebb03ff27c1d94077613b55^ -- /tmp/covid-19-public-data/data/covid-19_jhu-csse
  stderr: 'fatal: Could not switch to '/tmp/covid-19-public-data/': No such file or directory
'
```

## Additional context
Add any other context about the problem.

**Comment by emmjab** [comment]
hi @nperraud , thanks for the issue! Can you also paste in the line you called that resulted in the traceback?

Thanks!

**Comment by rokroskar** [comment]
Thanks for reporting @nperraud - this is due to a regression we had in `0.10.0` and it's been fixed in `0.10.2` but we need to do some interventions to fix the repository itself.

**Comment by rokroskar** [comment]
closing for now. this is not longer an issue in general.
